src/Features/CSharp/Portable/CodeRefactorings/CSharpRefactoringHelpersService.cs

```diff
@@ -7,13 +7,19 @@
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.LanguageServices;
 
+#nullable enable
```

Comment:
I think we've been putting these before the using block. Even if not that, blank line after.

Reply:
Yep, checked how it's done elsewhere, fixed.